.evergreen/config.yml

```diff
@@ -555,12 +555,16 @@ functions:
       params:
         working_dir: "src"
         script: |
-          # Disable xtrace (just in case it was accidentally set).
+          # Disable xtrace for security reasons (just in case it was accidentally set).
           set +x
+
           . ./prepare_atlas_connectivity.sh
           rm -f ./prepare_atlas_connectivity.sh
 
-          PYTHON_BINARY=${PYTHON_BINARY} bash ${PROJECT_DIRECTORY}/.evergreen/run-atlas-tests.sh
+          set -o errexit
+          set -o xtrace
+          export LC_ALL=en_US.UTF-8
```

Comment:
Can you add a comment for why this line is needed so that our future selves remember?